### DISCUSSION TITLE
Total Power and MQTT

### DISCUSSION THREAD

**Discussion by redzikx** [tbnobody/OpenDTU#353] [Ideas]
Hi Everyone!
I'm using OpenDTU  and it is working very well. Thank You for Your work!

I see that new version has on webui and it has additional fields that are global values (when you have many Inwerter this are sums) Total Power and Total Yield Day.

It would be nice to serve this data via MQTT. Pls consider this as new feature request.

**Comment by tbnobody** [comment]
#350 ...
